wchat/services/ChatService.js

```diff
@@ -1,7 +1,17 @@
 import axios from '../config/api.js'
 
-export function getChats() {
-  return axios.get('messages').then((response) => {
+export function getChats(ownerId, contactId) {
+  const path = 'messages?senderId=' + ownerId +
```

Comment:
Instead of concatenating string you can send the request params just like you did below:
```
 axios.get('messages', { params: { groupId } })
```